framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2545,7 +2545,7 @@ void VulkanReplayConsumerBase::ModifyCreateInstanceInfo(
     }
 
     // Enable validation layer and create a debug messenger if the enable_validation_layer replay option is set.
-    VkDebugUtilsMessengerCreateInfoEXT messenger_create_info{};
+    static VkDebugUtilsMessengerCreateInfoEXT messenger_create_info{};
```

Comment:
Can this be added to CreateInstanceInfoState?  Then this function doesn't have to concern itself with the lifetime of this struct.